cmd/logcli/main.go

```diff
@@ -157,6 +157,7 @@ func newQuery(instant bool, cmd *kingpin.CmdClause) *query.Query {
 		cmd.Flag("since", "Lookback window.").Default("1h").DurationVar(&since)
 		cmd.Flag("from", "Start looking for logs at this absolute time (inclusive)").StringVar(&from)
 		cmd.Flag("to", "Stop looking for logs at this absolute time (exclusive)").StringVar(&to)
+		cmd.Flag("step", "Query resolution step width").Default("1s").DurationVar(&query.Step)
```

Comment:
Not sure this is a good default for an hour window. Can you make it 30s ?

Reply:
Sure, I do understand your concerns. However, `1s` is the API default, reason why I kept it here. Can you confirm we prefer a different default here, please?

Reply:
Well that's a mistake I've made :) I think 15s or 30s would be better default even for the API.

Right now you get 3600 points if you have constant logging. I think it's too much.

I guess we can create an issue to change that later then ?

Reply:
Prometheus UI seems to default to 14s for an hour.

Reply:
I've changed to `30s`.

> Prometheus UI seems to default to 14s for an hour.

Are you suggest to keep a behaviour similar to Prometheus? If so, I can check the sources and see how it works there.

Reply:
Yes please ! Feel free to change our API default too at the same time.

Reply:
The Prometheus API requires the `step` parameter, while `promtool` calculates it with the same formula I've now introduced in the API for the default value. The idea of the last change set is that the default should be only in the API, while `logcli` simply doesn't pass the `step` param if it has not been explicitly specified on the cli via `--step`.

What's your take?

Reply:
awesome !